### PR TITLE
feat: Toolbox logo and documentation update

### PR DESCRIPTION
**Pull request by tbwiss**
# Description

This PR replaces the current icon and favicon with the newly designed logo for the Toolbox.
Several icon sizes and formats are provided when it is needed for different purposes (like for the OpenGraph and Twitter meta data).
Also, updates to the README and the documentation page are included plus an updated image of the query editor view.

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-graphql-docs-1885.surge.sh

File: .changeset/spotty-adults-leave.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"graphql-manual": patch
+"@neo4j/graphql-toolbox": patch
+"@neo4j/graphql": patch
```
Comment (darrellwarde) [inline .changeset/spotty-adults-leave.md:4]:
```suggestion
"@neo4j/graphql-toolbox": patch
```

This should really just be a bump to the Toolbox, we don't version the docs right now and there's no change to the library here!

File: packages/graphql-toolbox/src/modules/Login/Login.tsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ export const Login = () => {
         <div data-test-login-form className="grid place-items-center h-screen n-bg-neutral-90">
             <div className="w-login flex flex-col align-center justify-center bg-white shadow-md rounded p-8">
                 <div className="mb-6 text-center">
-                    <img src={Icon} alt="Neo4j Logo" className="h-12 w-12 mx-auto" />
+                    <img src={Icon} alt="Neo4j Logo" className="h-12 w-12 mb-3 mx-auto" />
```
Comment (darrellwarde) [inline packages/graphql-toolbox/src/modules/Login/Login.tsx:65]: I'm not sure how I feel about there being zero Neo4j branding on the login page - what do other tools do?

Comment (tbwiss) [reply packages/graphql-toolbox/src/modules/Login/Login.tsx:65]: Yes good point, I had mixed feeling for this particular change. I quickly checked a few other products:

https://browser.neo4j.io/ - No logo in the login modal
https://bloom.neo4j.io/ - No logo in the login card (Neo4j logo on the bottom left)
https://data-importer.neo4j.io/ - Neo4j logo in the login modal

I'll get the Neo4j logo back for our login modal

**Review by mjfwebb — APPROVED** [review]
Good work! Nice new logo :)

**Comment by tbwiss** [comment]
@darrellwarde do you think this is good to go now after bringing back the Neo4j logo to the login modal?

**Review by darrellwarde — APPROVED** [review]

**Pull request merged**